### DISCUSSION TITLE
Как корректно завершать работу spoof-dpi под macos?

### DISCUSSION THREAD

**Discussion by xalleop** [xvzc/SpoofDPI#125] [Q&A] [answered]
Как корректно завершать работу spoof-dpi под macos? Если прервать его работу в терминале клавишами Ctrl-Z, то после этого никакие сайты не грузятся. Если снова запустить spoof-dpi, то результат такой:  
[PROXY] Error creating listener: listen tcp4 127.0.0.1:8080: bind: address already in use - и по преднему сайты не грузятся.
Только если принудительно (Force Quit ) завершить процесс через Activity Monitor, он останавливается, но  соединения с сайтами нет.  И что делать? Есть ли способ корректно завершать его работу без Force Quit?

**Comment by PoorMouse** [comment, marked as answer]
Я завершаю по Ctrl+C. Потом закрывают консоль. Никаких проблем с открытием сайтов нет.

**Reply by xalleop** [reply]
Thanks, it worked.
